packages/rich-text/src/get-text-content.js

```diff
@@ -2,7 +2,8 @@
  * Get the textual content of a Rich Text value. This is similar to
  * `Element.textContent`.
  *
- * @param {Object} value Value to use.
+ * @param {Object} value      Value to use.
+ * @param {string} value.text Text contents.
```

Comment:
Do you think we could create a type for rich text values and then use it here and in other places?

Reply:
Sorry @ellatrix . I didn't get your point. Can you please explain it?

> create a type for rich text values and then use it here and in other places.

Reply:
@ellatrix Do you mean something like this?

```
@typedef value
@param {value} text Text Content.
```